### PR TITLE
remove jade from prerelease website

### PR DESCRIPTION
**Pull request by wjwwood**
I left the images because the other old distro images are still there too. Not sure if we should remove them or not, I guess we could.

### CODE REVIEW

File: prerelease_website/static/css/index.css [hunk 1]
```diff
@@ -51,20 +51,6 @@ html, body {
   color: #46B6EF;
 }
 
-.jade-column
-{
-  height: 100%;
-  background: #19A36A;
-  width: 25%;
```
Comment (mikaelarguedas) [inline prerelease_website/static/css/index.css]: Do we need to resize the other column to 33% if we go back to only 3 columns ?
(I didnt try to deploy the change so maybe it's not necessary)

Comment (wjwwood) [reply prerelease_website/static/css/index.css]: Not sure, I didn't try it either. I'll check locally and see.

Comment (wjwwood) [reply prerelease_website/static/css/index.css]: I didn't see an issue with the 25%, but it also doesn't make sense and looks ok with 33%, so I changed it.

**Comment by wjwwood** [comment]
Deployed.

**Pull request merged**